README.md

```diff
@@ -1,2 +1,34 @@
 # Project-Microverse
-First project for Microverse week I
+First project for Microverse week I.
+## Built With a html, css, gitignore and workflows GitHub.
+## Live Demo: http://127.0.0.1:5500/index.html
+## Getting Started: To get a local copy up and running, follow these steps.
+### Prerequisites
+* git
+* github
+* vscode
+* google chrome.
```

Comment:
> Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

- [ ] According to the project requirement which I also pointed out above, you are meant to use the Readme template provided by Microverse and then customize each section to match your project. :+1: :100:  I suggest you visit [this link](https://github.com/microverseinc/readme-template) and get the template and use it in this your project :+1: :smile: